test/extensions/filters/udp/dns_filter/dns_filter_test.cc

```diff
@@ -6,6 +6,7 @@
 #include "test/mocks/event/mocks.h"
 #include "test/mocks/server/mocks.h"
 #include "test/test_common/environment.h"
+#include "test/test_common/simulated_time_system.h"
```

Comment:
I looked for a test case with the clusters and didn't find one

Reply:
I have one.  I didn't include it here to reduce the PR size.   Let me see what I can do.